packages/taco-auth/src/types.ts

```diff
@@ -19,10 +19,17 @@ export interface AuthSignature {
   typedData: EIP712TypedData | EIP4361TypedData;
 }
 
+/**
+ * @deprecated Use EIP4361_AUTH_METHOD instead.
+ */
 export const EIP712_AUTH_METHOD = 'EIP712';
 export const EIP4361_AUTH_METHOD = 'EIP4361';
 
 export const USER_ADDRESS_PARAM_DEFAULT = ':userAddress';
+
+/**
+ * @deprecated Use USER_ADDRESS_PARAM_EIP4361 instead.
```

Comment:
These context variables were never publicly released - just the epic, right? If so, then a removal can be done instead of deprecation.

Reply:
They were never exposed to the `taco` API, but the values they represent were in use, hence they should be deprecated.

Reply:
No, I think you are right. I think that was my mistake. I will fix it.

Reply:
^ Is the goal to remove these additional context variables in a separate PR then?

Reply:
Yes, there's a follow-up issue for that: https://github.com/nucypher/taco-web/issues/536

Reply:
But these don't need to wait T+2 releases to be removed ... ?